app/components/common/userInput/index.test.tsx

```diff
@@ -1,53 +1,71 @@
 import { fireEvent, render, screen } from '@testing-library/react';
 import UserInput from '.';
 
-it('renders a input field', () => {
-  render(
-    <UserInput
-      label="Username"
-      placeholder="enter your name"
-      link="hap.day/"
-      value=""
-      setValue={() => {}}
-    />
-  );
-  const userInput = screen.getByTestId('user-input');
-  expect(userInput).toBeInTheDocument();
-});
+describe('UserInput', () => {
+  it('renders a input field', () => {
+    render(
+      <UserInput
+        label="Username"
+        placeholder="enter your name"
+        link="hap.day/"
+        value=""
+        setValue={() => {}}
+      />
+    );
+    const userInput = screen.getByTestId('user-input');
```

Comment:
In some tests `getBy*` by destructuring `render()` is used and in some test `screen`, would recommend using only one for consistency.

Reply:
It is the old code and has reused the same part of logic which was present. Will replace.